GitUI/UserControls/FileStatusList.cs

```diff
@@ -552,12 +553,12 @@ public GitItemStatus SelectedItem
 
         [DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)]
         [Browsable(false)]
-        public string SelectedItemParent
+        public GitRevision SelectedItemParent
         {
             get
             {
                 foreach (ListViewItem item in FileStatusListView.SelectedItems)
-                    return (string)item.Group?.Tag;
+                    return (GitRevision)item.Group?.Tag;
```

Comment:
I am not sure this is completely safe cast
Perhaps this is better: `return item.Group?.Tag as GitRevision;`?

Reply:
TAG is (currently) just set as a GitRevision, now always set. But I agree the cast should be changed, it was just a carry over.

Reply:
As unlikely as it may be, if in some usecase Tag is set to something else - it will break but it may not be apparent straight away...